src/tests/unit/test_tree_functionality/test_tree_node_update.py

```diff
@@ -126,6 +129,23 @@ def get_blueprint(type: str):
                     ],
                 }
             )
+        elif type == "FormBlueprint":
```

Comment:
new blueprint is added, that is used in the new test "test_set_update_uncontained_child"